base/linalg/triangular.jl

```diff
@@ -1900,14 +1907,10 @@ end
 function sqrtm{T}(A::UnitUpperTriangular{T})
```

Comment:
I'm lost. I see a small improvement, ~10%, doing this for the `UnitUpperTriangular` method, and it now has the same performance as converting to `UpperTriangular` and back. I see no improvement at all for `UpperTriangular` methods. But all the array access is already wrapped in `@inbounds` anyway, so is there really still `i<=j` testing going on? And `@time`ing `A[i,j]` versus `B[i,j]` shows no difference, no matter the type of `A` ...